### ISSUE TITLE
Elevated plasma saccharopinemia level 

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7084]
**Preferred term label:**
Elevated plasma saccharopinemia level 

**Definition (free text, please give PubMed ID)**
Abnormal level of saccharopinemia  in the plasma serum 
PMID: 33996359_30573525_32768327

**Parent term (use hpo.jax.org/app)**
Abnormal circulating lysine concentration HP:0010908

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Saccharopinuria, ORPHA:3124, OMIM: 268700

**Comment by pnrobinson** [comment]
Note saccharopinemia means elevated saccharopine in the blood circulation.

Adding the term as
HP:0034026
Elevated circulating sacchoropine concentration
Def: An increased concentration of saccharopine in the blood circulation. L-saccharopine is the N(6)-(1,3-dicarboxypropan-1-yl) derivative of L-lysine.
PMID: 571908
